src/sidebar/AccountMenuItem.js

```diff
@@ -17,7 +17,8 @@ export default class AccountMenuItem extends PureComponent {
 		} else if (window.location.href.indexOf('/beta') === -1) {
```

Comment:
@brucebinary How about using `window.BinaryBoot.isBeta` ?

Reply:
Ops. I'll have that changed.